WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingAddresses/WooShippingEditAddressViewModel.swift

```diff
@@ -383,6 +393,35 @@ final class WooShippingEditAddressViewModel: ObservableObject, Identifiable {
             }
         }
     }
+
+    /// Updates the destination address remotely with the provided (normalized) address and other edits.
+    private func updateConfirmedDestinationAddress(for orderID: Int64,
+                                                   with address: WooShippingAddress) {
+        // Merge the provided (normalized) address with the edited address fields.
+        let address = WooShippingDestinationAddress(company: address.company,
+                                                    address1: address.address1,
+                                                    address2: address.address2,
+                                                    city: address.city,
+                                                    state: address.state,
+                                                    postcode: address.postcode,
+                                                    country: address.country,
+                                                    phone: address.phone,
+                                                    name: name.value,
```

Comment:
Should we use `address.name` for this now that we have the Networking layer fixes for getting the name from the response (name vs. first + last name)?

Reply:
Good point. Updated in https://github.com/woocommerce/woocommerce-ios/pull/15260/commits/5cb03957f3372c23c56e308710979b2a252f5e48